externals/vjson/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+project(VJSON)
```

Comment:
The `configureHook` and `makeHook` from the src subdirectory should be removed

Reply:
Will do.